diff --git a/dist/types/public/index.d.ts b/dist/types/public/index.d.ts
index 49deb9f1fd0ad13cdd773ed3157fd74104307c68..602402fd841dc080bd7e3e9b14cf5ea5c458a0fd 100644
--- a/dist/types/public/index.d.ts
+++ b/dist/types/public/index.d.ts
@@ -13,7 +13,7 @@ export type * from './manifest.js';
 export type { AstroIntegrationLogger } from '../../core/logger/core.js';
 export type { ToolbarServerHelpers } from '../../runtime/client/dev-toolbar/helpers.js';
 export type { RemotePattern } from '@astrojs/internal-helpers/remote';
-export type { MarkdownHeading, RehypePlugins, RemarkPlugins, ShikiConfig, } from '@astrojs/markdown-remark';
+export type { MarkdownHeading, RehypePlugins, RemarkPlugins, ShikiConfig, RehypePlugin, RemarkPlugin } from '@astrojs/markdown-remark';
 export type { ExternalImageService, ImageService, LocalImageService, } from '../../assets/services/service.js';
 export type { GetImageResult, ImageInputFormat, ImageMetadata, ImageOutputFormat, ImageQuality, ImageQualityPreset, ImageTransform, UnresolvedImageTransform, } from '../../assets/types.js';
 export type { AssetsPrefix, SSRManifest } from '../../core/app/types.js';
